bin/luigi

```diff
@@ -1,6 +1,14 @@
 #!/usr/bin/env python
 
-import luigi
+import sys
+import warnings
+import luigi.cmdline
+
+
+def main(argv):
+    warnings.warn("'bin/luigi' has moved to console script 'luigi'", DeprecationWarning)
```

Comment:
@Tarrasch, I was again deprecation :P , that was part of #839. I don't think is in the doc, but once you have installed Luigi, the command should be in your path.

``` bash
virtualenv luigi_env
. luigi_env/bin/activate
cd luigi
python setup.py install
luigi someTask
```